YoCode/UnitConverterCheck.cs

```diff
@@ -85,95 +130,66 @@ public List<string> ExtractActions(List<string> actionLines)
             return list;
         }
 
-        public void InitializeLists()
-        {
-            actual = new List<UnitConverterResults>();
-            texts = new List<string> { "5", "25", "125" };
-            expectedOutputs = GetExpectedOutputs();
-            actions = GetActions(HTMLcode);
-        }
-
-        public async Task<List<UnitConverterResults>> ExecuteTheCheck()
+        public List<double> MakeConversion(List<string> inputs, double mult)
         {
-            for (int i = 0; i < texts.Count; i++)
+            var list = new List<double>();
+            foreach (string x in inputs)
             {
-                UnitConverterResults tempActual = new UnitConverterResults();
-                tempActual.input = texts[i];
-
-                for (int j = 0; j < actions.Count; j++)
-                {
-                    tempActual.action = actions[j];
-                    var formContent = GetEncodedContent(i, j);
-
-                    var bar = await client.PostAsync("/Home/Convert", formContent);
-
-                    tempActual.output = await GetResponseAsTaskAsync(bar);
-                    actual.Add(tempActual);
-                }
+                list.Add(Double.Parse(x) * mult);
             }
-            return actual;
-        }
-
-        private FormUrlEncodedContent GetEncodedContent(int i, int j)
-        {
-            return new FormUrlEncodedContent(new[]
-            {
-                        new KeyValuePair<string,string>("text",texts[i]),
-                        new KeyValuePair<string, string>("action",actions[j])
-                    });
-        }
-
-        private static Task<string> GetResponseAsTaskAsync(HttpResponseMessage bar)
-        {
-            return bar.Content.ReadAsStringAsync();
+            return list;
         }
 
-        public void GetResponseAsString(Task<string> bar)
+        public List<string> GetActionKeywords()
         {
-            tempResultStorage = bar.Result;
+            return new List<string> { "action", "value" };
         }
 
-        public List<double> ConvertAnswersToDouble(List<String> strings)
+        public List<double> CheckActions(string action)
         {
-            return strings.Select(s => double.Parse(s)).ToList();
+                foreach (var keywords in KeywordMap)
+                {
+                    if (action.ToLower().ContainsAny(keywords.Key))
+                    {
+                        return keywords.Value;
+                    }
+                }
+            return new List<double>{0.1};
         }
 
-        public List<string> GetActionKeywords()
+        public (double,double) StringToDouble(string a, string b)
         {
-            return new List<string> { "action", "value" };
+            return (Double.Parse(a), Double.Parse(b));
```

Comment:
It's more idiomatic to use the C# type names here rather than the CLR names, e.g. `double.Parse` instead of `Double.Parse`.